### PR TITLE
Update app insight dashboard query parameters

### PR DESCRIPTION
**Pull request by lamaks**
#### Description of changes

Update app insight dashboard query parameters to fix chart data

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)

### CODE REVIEW

File: packages/resource-deployment/templates/dashboard.template.json [hunk 1]
```diff
@@ -320,9 +320,9 @@
                                                             "resourceMetadata": {
                                                                 "id": "[variables('appInsightsResourceId')]"
                                                             },
-                                                            "name": "customEvents/custom/pendingScanRequests",
-                                                            "aggregationType": 4,
-                                                            "namespace": "microsoft.insights/components/kusto",
+                                                            "name": "pendingScanRequests",
+                                                            "aggregationType": 1,
```
Comment (pownkel) [inline packages/resource-deployment/templates/dashboard.template.json:324]: What is the new aggregation type?

Comment (lamaks) [reply packages/resource-deployment/templates/dashboard.template.json:324]: Sum

File: packages/resource-deployment/templates/dashboard.template.json [hunk 2]
```diff
@@ -461,9 +461,9 @@
                                                             "resourceMetadata": {
                                                                 "id": "[variables('appInsightsResourceId')]"
                                                             },
-                                                            "name": "customEvents/custom/totalScanRequests",
+                                                            "name": "totalScanRequests",
                                                             "aggregationType": 1,
```
Comment (pownkel) [inline packages/resource-deployment/templates/dashboard.template.json:465]: Should these next three metrics use sum as well?

Comment (lamaks) [reply packages/resource-deployment/templates/dashboard.template.json:465]: It does use sum. What metrics do you refer particularly?
`"aggregationType": 1`

Comment (pownkel) [reply packages/resource-deployment/templates/dashboard.template.json:465]: Sorry I misread, thought these were using average

**Pull request merged**